3/3-1.md

```diff
@@ -0,0 +1,98 @@
+```
+shortname: 3.1/UPSERT-VALIDATORS
```

Comment:
I understand why you want to number this 3.1: it replaces BEP-3, but BEPs have integer numbers.

Once this BEP is implemented, it will become stable, and then we can deprecate BEP-3 and add a note in the BEP-3 text telling readers that it was replaced by this BEP, which can have a number like BEP-20.

Reply:
> I understand why you want to number this 3.1: it replaces BEP-3, but BEPs have integer numbers.
> 
> Once this BEP is implemented, it will become stable, and then we can deprecate BEP-3 and add a note in the BEP-3 text telling readers that it was replaced by this BEP, which can have a number like BEP-20.

I'd add a hyperlink from BEP-3 to BEP-20. We can formalize this process in BEP-2.